test/lib/agent_helper.js

```diff
@@ -346,10 +340,6 @@ helper.flushRedisDb = (client, dbIndex) => {
   })
 }
 
-helper.withSSL = () => {
-  return Promise.all([fs.readFile(KEYPATH), fs.readFile(CERTPATH), fs.readFile(CAPATH)])
-}
```

Comment:
With the changes in this PR, there are no longer any references to this method. So, goodbye.